libs/s25main/network/GameClient.h

```diff
@@ -164,7 +172,9 @@ class GameClient final :
     void SystemChat(const std::string& text) override;
     void SystemChat(const std::string& text, unsigned char fromPlayerIdx);
 
-    void ToggleHumanAIPlayer();
+    /// @brief Toggle current player to be an AI player
+    /// @param aiInfo           AI to replace with
```

Comment:
Maybe shorten it to:
```suggestion
    /// Toggle current player to be an AI player of the given type
```

Reply:
agreed